src/locales/en.json

```diff
@@ -65,44 +63,27 @@
     }
   },
   "about": {
-    "title": "About Openverse",
+    "title": "About {openverse}",
     "description": {
-      "content": "Openverse is a tool that allows openly licensed and public domain works to be discovered and used by everyone."
+      "content": "{openverse} is a tool that allows openly licensed and public domain works to be discovered and used by everyone."
     },
     "collection": {
-      "content": "Openverse searches across more than 300 million images from open APIs and the {common-crawl} dataset. It goes beyond simple search to aggregate results across multiple public repositories into a single catalog, and facilitates reuse through features like machine-generated tags and one-click attribution.",
-      "common-crawl": "Common Crawl"
+      "content": "{openverse} searches across more than 300 million images from open APIs and the {common-crawl} dataset. It goes beyond simple search to aggregate results across multiple public repositories into a single catalog, and facilitates reuse through features like machine-generated tags and one-click attribution."
```

Comment:
This one also needs the variable added 

https://github.com/WordPress/openverse-frontend/blob/8a208a7e67dcfbcb40b57197782f07f240159aed/src/pages/about.vue#L13